applications/ThermalDEMApplication/CMakeLists.txt

```diff
@@ -3,14 +3,14 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 message("**** configuring KratosThermalDEMApplication ****")
 
 kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/DEMApplication)
-kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/DelaunayMeshingApplication)
+## kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/DelaunayMeshingApplication) ## UNCOMMENT TO GET TETGEN FROM DELAUNAY_MESHING APPLICATION
```

Comment:
maybe is it better to solve directly the DelaunayApp? @KratosMultiphysics/technical-committee